File: 프로그래머스/unrated/181865. 간단한 식 계산하기/간단한 식 계산하기.java
class Solution {
    public int solution(String binomial) {
   String[] parts = binomial.split(" ");
        int a = Integer.parseInt(parts[0]);
        int b = Integer.parseInt(parts[2]);
        
        return parts[1].equals("+") ? a + b : parts[1].equals("-") ? a - b : a * b;

    }
}

File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.py
def solution(binomial):
    arr = binomial.split(" ")
    answer = 0
    if arr[1] == '+':
        answer = int(arr[0]) + int(arr[2])
    elif arr[1] == '-':
        answer = int(arr[0]) - int(arr[2])
    else:
        answer = int(arr[0]) * int(arr[2])
    return answer